src/cpp/src/device_config.hpp

```diff
@@ -29,6 +29,20 @@ class DeviceConfig {
         if (m_device == "CPU") {
             auto inference_precision = core.get_property(device, ov::hint::inference_precision);
             m_kv_cache_type = inference_precision == ov::element::bf16 ? ov::element::bf16 : ov::element::f16;
+            // if user sets precision hint, kv cache type should be changed
+            if (plugin_config.find("INFERENCE_PRECISION_HINT") != plugin_config.end()) {
+                const auto& type_name = plugin_config.at("INFERENCE_PRECISION_HINT").as<std::string>();
```

Comment:
why don't we use properties and use hardcoded strings? Let's use OV entities

Reply:
@luo-cheng2021 ping

Reply:
Sorry for late, updated.